scripts/visualize/portfolio_sessions_all.R

```diff
@@ -54,29 +61,35 @@ visualize.portfolio_sessions_all <- function(viz=as.viz("portfolio_sessions_all"
           strip.background = element_blank(),
           axis.ticks=element_blank(),
           legend.position = "none"
-          ) 
-  
+          )
+
   info_graph <- ggplot_build(port_graph)
   layout_stuff <- info_graph$layout
-  lower_ranges <- layout_stuff$panel_ranges[[12]]$x.range
-  high_ranges <- layout_stuff$panel_ranges[[4]]$x.range
-  
+
+  if(packageVersion("ggplot2") >= "2.2"){
```

Comment:
But I guess we don't have a hard fail set up if the user doesn't have that version?